### DISCUSSION TITLE
How to solve the crash of the blank space behind the iPad click pre-input text

### DISCUSSION THREAD

**Discussion by Winstonokk** [singerdmx/flutter-quill#2141] [Mobile]
Steps to reproduce
1、Enter Chinese in the edit area, but do not confirm
2、Click on the blank space after the pre-entered text

https://github.com/user-attachments/assets/0cb90565-34bb-48e3-b0f0-7777cd1ff61a

is there a good engineer to help look at this problem? thanks very much

**Comment by EchoEllet** [comment]
Can you share the log with the error?

**Comment by Winstonokk** [comment]
https://github.com/singerdmx/flutter-quill/issues/2135 I have posted a log in the details of this issue, please check it, thank you!



---- Replied Message ----
| From | ***@***.***> |
| Date | 09/14/2024 20:37 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [singerdmx/flutter-quill] How to solve the crash of the blank space behind the iPad click pre-input text (Discussion #2141) |

Can you share the log with the error?
